views/error404.html

```diff
@@ -0,0 +1,15 @@
+% include('base.html', title="PyJudge")
+<body>
+	<div id="notfound">
```

Comment:
all this extra classes are necessary?

Reply:
this classes are for styling and responsive purpose only

Reply:
oh. you mean without the nested `notfound` classes/ids we can't make it responsive? I don't know enough about HTML/CSS but this code seems too much effort for a 404 page